java/api/src/main/java/io/ray/api/options/ActorCreationOptions.java

```diff
@@ -31,8 +31,13 @@ private ActorCreationOptions(Map<String, Double> resources, int maxRestarts,
     private String jvmOptions = null;
     private int maxConcurrency = 1;
 
+    public Builder setResource(String key, Double value) {
```

Comment:
```suggestion
    public Builder setResource(String resourceName, Double resourceQuantity) {
```
Document this: `Set a resource to reserve for the lifetime of this actor. This method can be called multiple times. If the same resource is set multiple times, the latest quantity will be used.`